data/dev/analysis/filters/filter005/rubrics/_filters_rubric_template.yaml

```
@@ -0,0 +1,5 @@
+# Common rubric variables for filter005.
+topology_target: Active cascaded low-pass (two second-order sections)
+topology_rejects: notch, all-pass
+grounded_guidance: Cite active elements (op-amps), R/C elements, and nodes that form the filter stages.
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)